spec/src/modules/quizzes.js

```diff
@@ -179,15 +200,6 @@ describe(`ConstructorIO - Quizzes${bundledDescriptionSuffix}`, () => {
       return expect(quizzes.getQuizNextQuestion(null, {})).to.eventually.be.rejected;
     });
 
-    it('Should be rejected if an invalid versionId is provided', () => {
```

Comment:
Hm.. we might want to keep this test around, but adjust the test to use `quizVersionId` instead. However, it looks like `quiz_version_id` isn't working the same as `version_id`. 

For example:

https://quizzes.cnstrc.com/v1/quizzes/farmstand-coffee-quiz/next/?key=key_aXLmVpkVp4BX21Sw&a=seen&a=true&a=seen&a=1&a=2&a=seen&a=1&quiz_version_id=123

vs

https://quizzes.cnstrc.com/v1/quizzes/farmstand-coffee-quiz/next/?key=key_aXLmVpkVp4BX[…]a=seen&a=true&a=seen&a=1&a=2&a=seen&a=1&version_id=123

Reached out to Vishnu on [Slack](https://constructor.slack.com/archives/C04M09VKNNB/p1680292190255649?thread_ts=1679436486.711039&cid=C04M09VKNNB)

Reply:
Looks like this has been updating and they should work the same way. Let's bring the test back and update it. 👍